test/manual/test_tracing.py

```
@@ -65,22 +72,33 @@ def __clear_trace_file(self):
         except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)